### PR TITLE
OHM-279: Add view for transaction error details

### PR DESCRIPTION
**Pull request by hnnesv**
screen shot 2016-08-05 at 14 41 44
screen shot 2016-08-05 at 14 41 55

### CODE REVIEW

File: app/views/transactionDetails.html [hunk 1]
```diff
@@ -180,6 +183,11 @@ <h6 class="subtitle"><em>Received at {{transactionDetails.response.timestamp | d
                     <div class="hljsFormat" hljs source="transactionDetails.response.body"></div>
                     <button style="margin-top: 10px;" class="btn btn-primary pull-right" ng-click="viewBodyDetails('Response', transactionDetails.response.body, transactionDetails.response.headers);">View Full Response Body</button>
                   </div>
+                  <div ng-if="transactionDetails.error" style="display: inline-block; width: 100%; margin-top: 15px;">
+                    <h4>Error</h4>
+                    <div>{{transactionDetails.error.message}}</div>
```
Comment (napiergit) [inline app/views/transactionDetails.html]: Maybe this should be the same as line 183? <div class="hljsFormat" hljs source="transactionDetails.error.message"></div>

File: app/views/transactionsAddReqResModal.html [hunk 2]
```diff
@@ -96,6 +97,11 @@ <h6 class="subtitle"><em>Received at {{record.response.timestamp | date:'yyyy-MM
             <div class="hljsFormat" hljs source="record.response.body"></div>
             <button style="margin-top: 10px;" class="btn btn-primary pull-right" ng-click="toggleFullView('Response', record.response.body, record.response.headers);">View Full Response Body</button>
           </div>
+          <div ng-if="record.error" style="display: inline-block; width: 100%; margin-top: 15px;">
+            <h4>Error</h4>
+            <div>{{record.error.message}}</div>
```
Comment (napiergit) [inline app/views/transactionsAddReqResModal.html]: Same here

**Comment by napiergit** [comment]
LGTM @hnnesv, just the two minor comments above

**Comment by rcrichton** [comment]
Just a quick minor comment from me. Looks great, just wondering if we should explain why an error is reported. Maybe call it Internal Error with a tooltip ? of 'An internal error occurred that prevented the OpenHIM form being able to forward a request'.

**Comment by hnnesv** [comment]
Cool, added the tooltip

**Pull request merged**